packages/@sanity/field/src/diff/components/ChangeBreadcrumb.tsx

```diff
@@ -21,11 +20,19 @@ export function ChangeBreadcrumb({
 
         return (
           <Fragment key={idx}>
-            {idx > 0 && <em className={styles.change__breadcrumb__separator}> / </em>}
-            <ChangeTitleSegment change={change} segment={titleSegment} />
+            {idx > 0 && (
+              <Box paddingX={1}>
+                <Text size={1} muted as="em">
+                  /
+                </Text>
+              </Box>
+            )}
+            <Text size={1} weight="semibold">
```

Comment:
You did correct by converting `font-weight: 600` to `semibold`, but actually `medium` would be best so the these headers look similar to the field headers in the form.

Reply:
the field headers are also using semibold tho? 🤔 https://github.com/sanity-io/sanity/blob/9f38c5a26652a2ad7f1aa69405c724e23a1d60c6/packages/@sanity/base/src/components/formField/FormFieldHeaderText.tsx#L33

Reply:
Ah true, then why do they look lighter ... 🤔